### PR TITLE
Add new functions to website

### LINKED ISSUE #730: Add new functions to website

### LINKED ISSUE #734: [BUG] Website build fails locally
**Describe the bug**
When following the guide [here](https://vmweventbroker.io/kb/contribute-site) to build the website locally, the build fails with the message `/usr/local/lib/ruby/site_ruby/2.7.0/rubygems.rb:277:in `find_spec_for_exe': Could not find 'bundler' (1.17.3) required by your /srv/jekyll/Gemfile.lock. (Gem::GemNotFoundException)`

**To Reproduce**
1. Follow the guide located [here](https://vmweventbroker.io/kb/contribute-site)
2. Build fails with error as above

**Expected behavior**
Expect the build to work without errors. Tagging the Jekyll docker image with `3.8` fixes this issue, so suspect something has changed in `jekyll/jekyll:4.0.0` and later.

**Screenshots**

**Version (please complete the following information):**
 - VEBA Form Factor: N/A
 - VEBA Version: v0.7.0

**Additional context**

### PR DESCRIPTION
**Pull request by dsbibby**
Signed-off-by: David Bibby <davidsbibby@gmail.com>

## Summary

Please describe your changes and what you plan to introduce as part of this pull request submission. If it fixes a bug or resolves a feature request, be sure to link to the issue numbers following the template below.

## Pull Request Checklist
🚨 Please review the [guidelines for contributing](https://vmweventbroker.io/community) to this repository.

- [x] Please ensure that you are making a pull request against the **Development** branch
- [ ] Please use the `WIP` keyword in the title of your PR if you are not ready for review
- [x] Please ensure that you have opened a Github Issue if you are resolving/fixing a problem
- [x] Please ensure that you have [signed](https://help.github.com/en/github/authenticating-to-github/signing-commits) all commits and that you have [squashed](https://medium.com/@slamflipstrom/a-beginners-guide-to-squashing-commits-with-git-rebase-8185cf6e62ec) all relevant commits related to your change
- [x] Please make sure that you have tested your change locally by successfully [building and deploying the VMware Event Broker Appliance](https://vmweventbroker.io/kb/contribute-appliance) and/or [building and deploying VMware Event Router](https://vmweventbroker.io/kb/contribute-eventrouter)
- [ ] Please include any relevant screenshots and/or output as part of your testing
- [ ] Please include any documentation updates that is applicable for your changes

## Change Type

What types of changes does your code introduce to the VMware Event Broker Appliance?

_Put an `x` in all boxes that apply_

Please check the type of change your PR introduces:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation changes
- [ ] Other (please describe):

## Resolved Issues

List of Issues closed or resolved by this PR. Add multiple `Closes` keyword followed by the issue number (e.g. Closes #ISSUE-NUMBER)

Closes #730 
Closes #734 

## Testing Verification

* Successfully built and tested website locally.

## Additional Information

* Note: cannot test links to new functions until they are merged in to master

If you have any questions/comments, feel free to reach out to team on Slack [#vcenter-event-broker-appliance](https://vmwarecode.slack.com/archives/CQLT9B5AA)

Thank you from the VEBA Team! 🥳

### CODE REVIEW

File: docs/kb/contribute-site.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ Step 2 - Run the following command to start the [Jekyll Docker container image](
 docker run --rm \
   --volume="$PWD:/srv/jekyll" \
   --publish 4000:4000 \
-  jekyll/jekyll \
+  jekyll/jekyll:3.8 \
```
Comment (embano1) [inline docs/kb/contribute-site.md:75]: Definitely a good idea, thx!

**Review by embano1 — DISMISSED** [review]
Can't thank you enough for this!

**Comment by embano1** [comment]
nit: can you change the commit prefix from `feat:` to `docs:` as it is merely a docs change. 🙏

**Comment by dsbibby** [comment]
> nit: can you change the commit prefix from `feat:` to `docs:` as it is merely a docs change. 🙏

Yup, that should be done now 😄

**Review by embano1 — APPROVED** [review]
LGTM

**Pull request merged**